docs/development/compare_atomic_files.ipynb

```diff
@@ -6,145 +6,8878 @@
    "source": [
```

Comment:
<p>Don't we want to use the AtomDataCompare class here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/carsus/pull/315/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Sure we can, but the atom data files I am using were generated at different points in time. They have different keys and I thought they would be better for exploring differences.